src/Analysis/GWASV2/SelectStudyPopulation/Utils/CohortDefinitions.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery, queryConfig } from 'react-query';
+import { Table, Spin } from 'antd';
+import { fetchCohortDefinitions } from '../../Shared/wizardEndpoints/cohortMiddlewareApi';
+import { useFetch, useFilter } from '../../Shared/formHooks';
+
+const CohortDefinitions = ({
+  sourceId, selectedCohort = undefined, handleCohortSelect, otherCohortSelected, searchTerm,
```

Comment:
is this component supposed to be generic, meaning it is not only going to be used in SelectStudyPopulation? If yes, then I propose moving it to a different folder. If no, then I propose making it specific to SelectStudyPopulation step and removing things like "otherCohortSelected".

Reply:
The var and prop "otherCohortSelected" has been removed.